configs/config_sandbox/esrf16_altmodels_study2/dataset-esrf16_altmodels_study2_CropCase5_TrnValCaseB.yaml

```diff
@@ -0,0 +1,60 @@
+dataset_config:
```

Comment:
just to be clear for future (when we forget) that no new datasets were created, let's comment out all scans and note where we copied from (below). this will not actually be run, as not having a train and val set would break the code currently. we should change this later to allow choosing existing dataset components in dataset_prep or in each workflow as command line input:
```
dataset_config:
  dataset_split:
#    train: [   # copied from dataset-esrf16_segV1_study1_CropCase5_TrnValCaseB/train
#      "8bit_AS4_S1_P1_L1_2560_1800_2160",
#      "8bit_AS4_S1_P1_L3_2560_1800_2160",
#      "8bit_AS4_S1_P1_L5_2540_1720_2160",
#      "8bit_AS4_CNT_S2_P1_L0_2560_1800_2160",
#      "8bit_AS4_CNT_S2_P1_L5_2560_1800_2160",
#      "8bit_AS4_CNT_S1_P1_L2_2560_1750_2160",
#      "8bit_AS4_CNT_S1_P1_L4_2560_1750_2160",
#      "8bit_AS4_CNT_S1_P1_L6_2560_1750_2160",
#      "THIN_REF_S2_P1_L1_2434_1547_2159",
#      "THIN_REF_S2_P1_L3_2496_1563_2159"
#    ]
#    validation: [   # copied from dataset-esrf16_segV1_study1_CropCase6_TrnValCaseB/validation
#      "8bit_AS4_S1_P1_L0_2560_1800_2160",
#      "8bit_AS4_S1_P1_L2_2560_1800_2160",
#      "8bit_AS4_S1_P1_L4_2560_1800_2160",
#      "8bit_AS4_S1_P1_L6_2560_1800_2160",
#      "8bit_AS4_CNT_S2_P1_L4_2560_1800_2160",
#      "8bit_AS4_CNT_S2_P1_L6_2560_1800_2160",
#      "8bit_AS4_CNT_S1_P1_L3_2560_1750_2160",
#      "8bit_AS4_CNT_S1_P1_L5_2560_1750_2160",
#      "THIN_REF_S2_P1_L0_2508_1551_2159",
#      "THIN_REF_S2_P1_L2_2484_1524_2159",
#      "THIN_REF_S2_P1_L4_2433_1533_2159"
#    ]
#    test: [   # copied from dataset-esrf16_segV1_study1_CropCase6_TrnValCaseA/test
#      "THIN_CNT_S2_P1_L0_2349_1578_2159",
#      "THIN_CNT_S2_P1_L2_2293_1581_2159",
#      "THIN_CNT_S2_P1_L3_2292_1566_2159",
#      "THIN_CNT_S2_P1_L4_2334_1578_2159",
#      "THIN_CNT_S2_P1_L5_2316_1542_2159",
#      "8bit_AS4_S2_P1_L0_2560_1750_2160",
#      "8bit_AS4_S2_P1_L4_2560_1750_2160",
#      "8bit_AS4_S2_P1_L5_2560_1750_2160",
#      "8bit_AS4_S2_P1_L6_2560_1750_2160"
#    ]
```